incubator/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/config/MqttKafkaBindingConfig.java

```diff
@@ -29,6 +30,7 @@ public class MqttKafkaBindingConfig
     public final KindConfig kind;
     public final MqttKafkaOptionsConfig options;
     public final List<MqttKafkaRouteConfig> routes;
+    public MqttKafkaSessionFactory.KafkaWillProxy willProxy;
```

Comment:
Suggest adding a blank line before this to separate from final fields, improves readability.